src/Auth/Actions/userActions.js

```diff
@@ -76,40 +69,45 @@ export const LoginUser = (credentials, history, setFieldError, setSubmitting) =>
                 const token = userData.data.token
                 localStorage.setItem('token', token)
                 history.push("/dashboard")
-                dispatch(loginUserSuccess(userData))
+                return dispatch => {
+                    dispatch(loginUserSuccess(data))
+                }
             }
             setSubmitting(false);
-        })
-        .catch(err => console.error(err));
+    } catch (error) {
+        throw new Error(error)
```

Comment:
you should dispatch the error to store and show to the user

Reply:
Ok Sir